sovrin_client/test/cli/test_nym.py

```diff
@@ -73,14 +61,16 @@ def getNymNotFoundExpectedMsgs(idr):
     return ["NYM {} not found".format(idr)]
 
 
-def testGetDIDWithoutAddingIt(be, do, philCli, abbrevIdr):
-    getNym(be, do, philCli, abbrevIdr,
-           getNymNotFoundExpectedMsgs(abbrevIdr))
+def testGetDIDWithoutAddingIt(be, do, philCli, trust_anchor_did_signer):
+    ensureConnectedToTestEnv(be, do, philCli)
+    getNym(be, do, philCli, trust_anchor_did_signer.identifier,
+           getNymNotFoundExpectedMsgs(trust_anchor_did_signer.identifier))
 
 
-def testGetCIDWithoutAddingIt(be, do, philCli, trustAnchorSigner):
-    getNym(be, do, philCli, trustAnchorSigner.identifier,
-           getNymNotFoundExpectedMsgs(trustAnchorSigner.identifier))
+def testGetCIDWithoutAddingIt(be, do, philCli, trust_anchor_cid_signer):
```

Comment:
Plenum still supports them. So testing them still makes sense. I've changed the CLI to guild you to use DID instead and will only generate key-identifiers using DID but the NYM transaction still supports them so testing that capability still makes sense.